python/tests/test_file_format.py

```diff
@@ -825,8 +827,9 @@ def verify_missing_fields(self, ts):
         with kastore.load(self.temp_file) as store:
             all_data = dict(store)
         for key in all_data.keys():
-            # We skip this key as it is optional
-            if "metadata_schema" not in key:
+            # We skip these keys as it is optional
+            if "metadata_schema" not in key and key != "metadata":
+                print(key)
```

Comment:
```suggestion
```

Reply:
Facepalm! So close to a "hole in one" PR. Ah well, will try again next time!